components/script_traits/script_msg.rs

```diff
@@ -83,8 +83,9 @@ pub enum ScriptMsg {
     /// A new load has been requested, with an option to replace the current entry once loaded
     /// instead of adding a new entry.
     LoadUrl(PipelineId, LoadData, bool),
-    /// Dispatch a mozbrowser event to the parent of this pipeline.
-    /// The first PipelineId is for the parent, the second is for the originating pipeline.
+    /// Dispatch a mozbrowser event to a given iframe,
+    /// or to the window if no subpage id is provided.
```

Comment:
It happens, e.g., when we generate an error caused by a panic but we don't have a pipeline to blame it on, e.g. a panic from a resource thread.